README.md

```diff
@@ -558,10 +558,21 @@ $job = $client->jobs()->show($jobId);
 Returns the job.
 
 ##### Wait for a job to finish
+This will periodically poll the job status until the job either finished or the maximum wait time was reached
 ```php
-$numberOfMinutesToWait = 3;
+$numberOfSecondsToWait = 30;
```

Comment:
I'd stick with 180 here to keep the default people see at 3 minutes which is a bit more realistic with some of the clone operations and such?

```suggestion
$numberOfSecondsToWait = 180;
```